packages/istanbul-lib-instrument/src/instrumenter.js

```diff
@@ -26,11 +26,8 @@ export function defaultOpts() {
         plugins: [
             'asyncGenerators',
             'bigInt',
-            /* Verify and add upon release of node.js 12:
-             * classProperties
-             * classPrivateProperties
-             * classPrivateMethods
```

Comment:
It seems classPrivateMethods are not supported in node.js v12.0.0.  Will be keeping an eye on the v8 release blog and add plugins when appropriate.